package.json

```diff
@@ -17,7 +17,8 @@
     "create-plugin": "backstage-cli create-plugin",
     "release": "if [ \"$(git symbolic-ref --short HEAD)\" = master ]; then echo \"don't try to release master\"; exit 1; else lerna version --no-push; fi",
     "lerna": "lerna",
-    "storybook": "yarn workspace storybook start"
+    "storybook": "yarn workspace storybook start",
+    "build-storybook": "yarn workspace storybook install && yarn workspace storybook build-storybook"
```

Comment:
Afaik this is the same as just running `yarn install && yarn workspace storybook build-storybook`, how come we want that here? ^_>

Reply:
Can't imagine service providers don't support yarn install + yarn build, it's a really common pattern O_o

Reply:
Was the best I could come up with to trigger a PR to check if it works. Also thought that having `yarn run build-storybook` lets us switch out the command without changing how netlify in this case runs the command. But I'll close it, seems to work.

Reply:
Yep I'm not against having build-storybook in the root pkg as long as it provides some value. Thinking that currently users should be using the storybook command instead, and CI can easily execute the command in the pkg.  Was more the `yarn install` I was questioning :grin: